### PR TITLE
DOMA-3287 update miniapp example

### PR DESCRIPTION
**Pull request by AleX83Xpert**
Minimal miniapp example. Show curent user and current organization on main page.
image

### CODE REVIEW

File: apps/miniapp/domains/common/clients/CondoBridge.tsx [hunk 1]
```diff
@@ -0,0 +1,121 @@
+import { Organization, User } from '@app/condo/schema'
```
Comment (akarjakin) [inline apps/miniapp/domains/common/clients/CondoBridge.tsx:1]: why `.tsx` ? there are no components here

File: apps/miniapp/domains/common/utils/oidcAuth.tsx [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+                })
+            }
+        }
+
+    }, [isAllLoaded, user, isAuthenticated, condoUser, condoOrganization])
+
+    return (
+        <OidcAuthContext.Provider value={{ user, isLoading: (isUserLoading || isOrganizationLoading) }}>
+            {
+                (isAllLoaded && isAuthenticated && user.id === condoUser.id && user.organizationId === condoOrganization.id)
```
Comment (akarjakin) [inline apps/miniapp/domains/common/utils/oidcAuth.tsx:48]: please move this calculation to variable

File: apps/miniapp/domains/common/utils/organization.tsx [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+const condoBridge = createCondoBridge({ receiverOrigin: condoUrl })
+const OrganizationContext = React.createContext<IUseOrganizationHookValue>({})
+
+const useOrganization = () => useContext(OrganizationContext)
+
+const OrganizationProvider = ({ children }) => {
+    let ret
+    const { result: organization, isLoading } = condoBridge.useOrganization()
+
+    if (isLoading || organization === undefined) {
```
Comment (akarjakin) [inline apps/miniapp/domains/common/utils/organization.tsx:21]: no need for `let` and `else`

```
    if (isLoading || organization === undefined) return <div>Getting organization...</div>
    if (organization === null) return <div>No organization found</div>
    
    return (
            <OrganizationContext.Provider value={{ organization, isLoading }}>
                {children}
            </OrganizationContext.Provider>
    )
```

File: apps/miniapp/domains/common/utils/organization.tsx [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+    const WithOrg = ({ organization, ...pageProps }) => {
+        return (
+            <OrganizationProvider>
+                <PageComponent {...pageProps} />
+            </OrganizationProvider>
+        )
+    }
+
+    // Set the correct displayName in development
+    if (process.env.NODE_ENV !== 'production') {
```
Comment (akarjakin) [inline apps/miniapp/domains/common/utils/organization.tsx:46]: reusable constant

File: apps/miniapp/pages/_app.tsx [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+}
+
+const MyApp = ({ Component, pageProps }) => {
+    const intl = useIntl()
+    dayjs.locale(intl.locale)
+
+    const LayoutComponent = Component.container || BaseLayout
+    return (
+        <IFrameWrapper withUser withOrganization>
+            <ConfigProvider locale={ANT_LOCALES[intl.locale || 'en']} componentSize={'large'}>
```
Comment (akarjakin) [inline apps/miniapp/pages/_app.tsx:33]: please move `ANT_LOCALES[intl.locale || 'en']` to variable for readability

**Review by akarjakin — APPROVED** [review]

File: apps/miniapp/domains/common/clients/CondoBridge.tsx (hunk 1 above)
Comment (AleX83Xpert) [reply apps/miniapp/domains/common/clients/CondoBridge.tsx:1]: ¯\\\_(ツ)_/¯

**Review by sitozzz — APPROVED** [review]

**Pull request merged**